tests/src/FunctionalJavascript/MediaAVPortalCreateContentTest.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\media_avportal\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+use Drupal\Tests\media\Functional\MediaFunctionalTestTrait;
+use Drupal\Tests\media\Traits\MediaTypeCreationTrait;
+
+/**
+ * Base class for Media AV Portal functional JavaScript tests.
+ */
+class MediaAVPortalCreateContentTest extends WebDriverTestBase {
+
+  use MediaFunctionalTestTrait;
```

Comment:
In one of the test methods please also check that our constraint works. Meaning that you provide a proper URL but with an invalid reference which would trigger the constraint to show the violation that the resource does not exist.